src/components/Squeak/components/auth/SignUp.tsx

```diff
@@ -12,6 +13,13 @@ type SignUpProps = {
 
 export const SignUp: React.FC<SignUpProps> = ({ buttonText = 'Sign up', onSubmit, setMessage }) => {
     const { signUp } = useUser()
+    const postHog = usePostHog()
+
+    useEffect(() => {
+        postHog?.setPersonProperties({
+            'Has visited sign up page': true,
+        })
+    }, [])
```

Comment:
This is a squeak signup page I think? So I don't think we need this here.

Reply:
Ah my bad, thought I removed this commit, will do so